source/fuzz/fuzzer_pass_add_dead_breaks.cpp

```diff
@@ -34,24 +34,39 @@ void FuzzerPassAddDeadBreaks::Apply() {
   // We consider each function separately.
   for (auto& function : *GetIRContext()->module()) {
     // For a given function, we find all the merge blocks in that function.
-    std::vector<uint32_t> merge_block_ids;
+    std::vector<opt::BasicBlock*> merge_blocks;
     for (auto& block : function) {
       auto maybe_merge_id = block.MergeBlockIdIfAny();
       if (maybe_merge_id) {
-        merge_block_ids.push_back(maybe_merge_id);
+        auto merge_block =
+            fuzzerutil::MaybeFindBlock(GetIRContext(), maybe_merge_id);
+        if (merge_block == nullptr) {
+          continue;
+        }
```

Comment:
If maybe_merge_id is non-zero, there is guaranteed to be a merge block.

So either replace this with an assertion that merge_block is not null (with a comment via && "..."), or just do: merge_blocks.push_back(fuzzerutil::MaybeFindBlock(...))

Reply:
Fixed in a04b185.